### ISSUE TITLE
The adaptivenes of v0.1.0

### ISSUE THREAD

**Issue by Albertmoiseev** [technocracy-world/crypto-ratel#7]
The app is resizeable only vertically. It is not possible to resize it horizontally.

**Comment by ivgrd** [comment]
Yes. It is expected behavior. We are planning to launch mobile apps and expect the interface to be consistent.
